src/Middleware/Diagnostics/src/DeveloperExceptionPage/IDeveloperPageExceptionFilter.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Diagnostics
+{
+    public interface IDeveloperPageExceptionFilter
```

Comment:
A few questions:
- Should this be in the abstractions package and be renamed to something more general? I don't want to give the impression that these are ASP.NET Core wide exception filters. These only apply to the developer exception page.
- Do we need something similar for the error handler? It already has a way to handle the exception and get at the error (via a feature).

Reply:
I don't think personally think there's much point to abstractions packages anymore. Open to ideas or discussion about the pros and cons, but since we don't ship packages I don't think there are many cons to either choice.

I think the name makes it really clear that it only applies to the developer exception page.

If we add it to the error handler, then it **does**  become ASP.NET wide. However I'm not sure we'd do that. The reason why we're doing this for developer exception page is that we want to plug in more default behaviours and don't want you to end up with 20 lines of code in startup for it. I'm not sure we'd add more default behaviours to error handler. Hearing arguments for that would change my mind 😁

Reply:
Agreed. I was just posing these questions regardless.